In [1]:
import pandas as pd

df_1 = pd.read_excel('09_MATRICULAS_ED_SUPERIOR_MAULE_2021.xlsx')
df_1

ID     GENERO  EDAD RANGO EDAD  AÑO INGRESO SEMESTRE INGRESO  \
0            2  Masculino    24   20 a 24          2017  Primer semestre   
1           20  Masculino    27   25 a 29          2020  Primer semestre   
2           69  Masculino    22   20 a 24          2017  Primer semestre   
3          101   Femenino    21   20 a 24          2021  Primer semestre   
4          111   Femenino    22   20 a 24          2018  Primer semestre   
...        ...        ...   ...        ...          ...              ...   
48463  1048430   Femenino    19   15 a 19          2021  Primer semestre   
48464  1048431  Masculino    19   15 a 19          2021  Primer semestre   
48465  1048446   Femenino    22   20 a 24          2018  Primer semestre   
48466  1048484  Masculino    22   20 a 24          2020  Primer semestre   
48467  1048490   Femenino    19   15 a 19          2020  Primer semestre   

                TIPO DE INSTITUCION                    NOMBRE DE INSTITUCION  \
0            Universidades Privadas            UNIVERSIDAD AUTONOMA DE CHILE   
1          Institutos Profesionales                           IP SANTO TOMAS   
2               Universidades CRUCH           UNIVERSIDAD CATOLICA DEL MAULE   
3            Universidades Privadas            UNIVERSIDAD AUTONOMA DE CHILE   
4               Universidades CRUCH           UNIVERSIDAD CATOLICA DEL MAULE   
...                             ...                                      ...   
48463  Centros de Formacion Tecnica                          CFT SANTO TOMAS   
48464        Universidades Privadas            UNIVERSIDAD AUTONOMA DE CHILE   
48465        Universidades Privadas  UNIVERSIDAD TECNOLOGICA DE CHILE INACAP   
48466           Universidades CRUCH           UNIVERSIDAD CATOLICA DEL MAULE   
48467        Universidades Privadas            UNIVERSIDAD AUTONOMA DE CHILE   

      ACREDITACION INSTITUCIONAL   PERIODO DE ACREDITACION  ...  \
0                     ACREDITADA  29/10/2019 AL 29/10/2023  ...   
1                     ACREDITADA  20/01/2021 AL 20/01/2026  ...   
2                     ACREDITADA  20/01/2021 AL 20/01/2026  ...   
3                     ACREDITADA  29/10/2019 AL 29/10/2023  ...   
4                     ACREDITADA  20/01/2021 AL 20/01/2026  ...   
...                          ...                       ...  ...   
48463                 ACREDITADA  20/12/2019 AL 20/12/2024  ...   
48464                 ACREDITADA  29/10/2019 AL 29/10/2023  ...   
48465                 ACREDITADA  27/11/2018 AL 27/11/2021  ...   
48466                 ACREDITADA  20/01/2021 AL 20/01/2026  ...   
48467                 ACREDITADA  29/10/2019 AL 29/10/2023  ...   

                NIVEL CARRERA  AREA CONOCIMIENTO  \
0      Carreras Profesionales              Salud   
1      Carreras Profesionales       Agropecuaria   
2      Carreras Profesionales         Tecnologia   
3      Carreras Profesionales              Salud   
4      Carreras Profesionales          Educacion   
...                       ...                ...   
48463       Carreras Tecnicas              Salud   
48464  Carreras Profesionales  Ciencias Sociales   
48465  Carreras Profesionales          Educacion   
48466  Carreras Profesionales       Agropecuaria   
48467  Carreras Profesionales  Ciencias Sociales   

      DURACION PLAN DE ESTUDIO (SEMESTRES)  \
0                                       10   
1                                        4   
2                                       12   
3                                       14   
4                                        9   
...                                    ...   
48463                                    5   
48464                                   10   
48465                                    8   
48466                                   10   
48467                                   10   

      DURACION PROCESO TITULACION (SEMESTRES)  \
0                                           1   
1                                           0   
2       

# La poblacion son las matriculas de educacion superior de la region del Maule.

In [2]:
import pandas as pd

muestra1 = df_1.shape[0] #numero de filas
muestra1

48468

## La muestra son 48468 matriculas de la region del Maule.

VARIABLE CUANTITATIVA DISCRETA

  # **1. La clasificacion de la variable 'EDAD' es cuantitativa continua**
  # **2. La clasificacion de la variable 'VALOR ARANCEL (PESOS)' es cuantitativa continua**
  # **3. La clasificacion de la variable 'DURACIÓN PLAN DE ESTUDIOS (SEMESTRES)' es cuantitativa discreta**
  # **4. La clasificacion de la variable 'GÉNERO' es cualitativa nominal**
  # **5. La clasificacion de la variable 'SEMESTRE INGRESO' es cualitativa ordinal**



In [ ]:
# Importar librería
import pandas as pd

df_resumen = pd.read_excel('BBDD (Velocidad).xlsx')

intervalos = 8

# Vamos a agregar una nueva columna, con la función cut()
df_resumen['Intervalos'] = pd.cut(df_resumen['Velocidad (Mbps)'], bins = intervalos,
                                  precision = 1, include_lowest=True)

# Frecuencias
f = df_resumen.groupby('Intervalos').size()
F = f.cumsum()
h = (f/40)*100
H = h.cumsum()

# Armar el DataFrame
tabla = pd.DataFrame({
    'Frecuencia absoluta': f,
    'Frecuencia relativa (%)': h,
    'Frecuencia absoluta acumulada': F,
    'Frecuencia relativa acumulada (%)': H
})

tabla